backends/graphics/opengl/opengl-graphics.cpp

```diff
@@ -1294,16 +1294,16 @@ void OpenGLGraphicsManager::handleResizeImpl(const int width, const int height)
 	// possible and then scale it to the physical display size. This sounds
 	// bad but actually all recent chips should support full HD resolution
 	// anyway. Thus, it should not be a real issue for modern hardware.
-	if (   overlayWidth  > (uint)OpenGLContext.maxTextureSize
-	    || overlayHeight > (uint)OpenGLContext.maxTextureSize) {
+	if (overlayWidth > (uint)OpenGLContext.maxTextureSize
+		|| overlayHeight > (uint)OpenGLContext.maxTextureSize) {
```

Comment:
Please revert the whitespace changes

```suggestion
	if (   overlayWidth  > (uint)OpenGLContext.maxTextureSize
	    || overlayHeight > (uint)OpenGLContext.maxTextureSize) {
```